ern-local-cli/src/index.ts

```diff
@@ -55,21 +57,36 @@ function showInfo() {
 }
 
 function showVersion() {
-  // get electrode-native local-cli version
-  if (config.get('platformVersion')) {
-    log.info(`ern-local-cli : ${config.get('platformVersion')}`)
-  }
-  // get electrode-native global-cli version
-  const packageInfo = JSON.parse(
-    execSync('npm ls -g electrode-native --json').toString()
-  )
-  if (packageInfo && packageInfo.dependencies) {
-    log.info(
-      `electrode-native : ${
-        packageInfo.dependencies['electrode-native'].version
-      }`
+  let packageInfo
+  try {
+    const pkg = path.join(
+      execSync('yarn global dir')
```

Comment:
will this work without yarn installed on the user's terminal?

Reply:
Yeah, it will still work since it's wrapped in try-catch (it will log at trace level if it errors).